content/news/core-monthly/2023/core-monthly-2023-01-01-2023-01-31.md

```diff
@@ -0,0 +1,617 @@
+* [#30657](https://github.com/PrestaShop/PrestaShop/pull/30657): Functional Tests : FO - 01. Thank you, [@Progi1984](https://github.com/Progi1984)
+* [#30651](https://github.com/PrestaShop/PrestaShop/pull/30651): Functional tests - Add new test 'Bulk enable, disable, duplicate and delete' in product page V2'. Thank you, [@nesrineabdmouleh](https://github.com/nesrineabdmouleh)
+### Web services
+* [#30293](https://github.com/PrestaShop/PrestaShop/pull/30293): Aaa/apis01 implement cqrs based api endpoint. Thank you, [@mflasquin](https://github.com/mflasquin)
+## Code changes in the ‘8.0.x’ branch 
+### Back office
+* [#30962](https://github.com/PrestaShop/PrestaShop/pull/30962): Module from theme with tab using SF routing fails after enabling the theme. Thank you, [@jolelievre](https://github.com/jolelievre)
+* [#30818](https://github.com/PrestaShop/PrestaShop/pull/30818): Fix access for read only permissions. Thank you, [@dingedi](https://github.com/dingedi)
+* [#30745](https://github.com/PrestaShop/PrestaShop/pull/30745): Load default language if employee id lang does not exist. Thank you, [@mflasquin](https://github.com/mflasquin)
+* [#30415](https://github.com/PrestaShop/PrestaShop/pull/30415): Fix applying special char on translations. Thank you, [@mflasquin](https://github.com/mflasquin)
+* [#29837](https://github.com/PrestaShop/PrestaShop/pull/29837): Avoid possibility to create product with combinations when combinations feature is disabled. Thank you, [@mflasquin](https://github.com/mflasquin)
+* [#29647](https://github.com/PrestaShop/PrestaShop/pull/29647): BO - Import files - Do not show generated csv files from import/csvfromexcel folder in history of uploaded files. Thank you, [@mflasquin](https://github.com/mflasquin)
+* [#29488](https://github.com/PrestaShop/PrestaShop/pull/29488): Module translations no longer take translations from another lang if its default one.. Thank you, [@margud](https://github.com/margud)
+* [#29358](https://github.com/PrestaShop/PrestaShop/pull/29358): Fix price formatting in MoneyType. Thank you, [@zuk3975](https://github.com/zuk3975)
+### Core
+* [#31006](https://github.com/PrestaShop/PrestaShop/pull/31006): Fix conversion of hex color codes. Thank you, [@lmeyer1](https://github.com/lmeyer1)
+* [#30834](https://github.com/PrestaShop/PrestaShop/pull/30834): Fix SQL error introduced by da10f001. Thank you, [@lmeyer1](https://github.com/lmeyer1)
+### Front office
+* [#30992](https://github.com/PrestaShop/PrestaShop/pull/30992): Fix customer isLogged property in FO. Thank you, [@Hlavtox](https://github.com/Hlavtox)
+### Web services
+* [#30993](https://github.com/PrestaShop/PrestaShop/pull/30993): Fix issue #26568 about customer blank schema. Thank you, [@ceer](https://github.com/ceer)
+
+
+## Code changes in modules, themes and tools
+
+### .github
```

Comment:
Leaving a comment - should it be .github or GitHub ? Just wondering

Reply:
it's a directory name, it's ok to have it lowercase

Reply:
Ok that makes sense, thanks for the reply.